target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -3,6 +3,10 @@
 PART_NAME=firmware
 REQUIRE_IMAGE_METADATA=1
 
+get_magic_long_vr2600v() {
```

Comment:
I tested it before I committed the change. The metadata check was already active before this function was emptied. So I had to take care of it beforehand. So just one check was removed. Had been two before.